Source/SpectrumAnalysis/KTSequentialTrackFinder.cc

```diff
@@ -550,49 +550,53 @@ namespace Katydid
      }
 
 
-    bool KTSequentialTrackFinder::EmitPreCandidate(KTSequentialLineData line)
+    bool KTSequentialTrackFinder::EmitPreCandidate(KTSequentialLineData& line)
     {
         KTDEBUG(stflog, "applying cuts and then emitting candidate");
         bool lineIsCandidate = true;
 
+        line.CalculateTotalPower();
+        line.CalculateTotalSNR();
+        line.CalculateTotalNUP();
+
         if ( fApplyTotalPowerCut )
         {
-            if ( line.GetAmplitudeSum() <= fTotalPowerThreshold )
+            if ( line.GetTotalWidePower() <= fTotalPowerThreshold )
```

Comment:
no, you re right. I removed the cuts from the otc and the itc processors. I left them here, because the sequential track finder is the place where the lines are produced and maybe not all lines need to be emitted to the next processing steps.  The only good reason though to not remove the cuts here is that the apply-cuts can only apply the same cut once in the entire processing chain (that's what I understood). I can remove them if you like.

Reply:
ok! I think that we actually can have several cuts (this is what I recall from a discussion with Noah last week). Maybe @nsoblath can comment more on this...